index.html

```diff
@@ -1,74 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Bungee+Spice&display=swap"
-      rel="stylesheet"
-    />
-    <link
-      href="https://fonts.googleapis.com/css2?family=Chakra+Petch:ital,wght@1,500&display=swap"
-      rel="stylesheet"
-    />
+
+    <!-- Icons -->
```

Comment:
Good use of commenting for readability, something I should think about doing more often!